libraries/standard/mqtt/src/mqtt.c

```diff
@@ -41,6 +41,18 @@
     #define MQTT_MAX_CONNACK_RECEIVE_RETRY_COUNT    ( 5U )
 #endif
 
+/**
```

Comment:
The description for MQTT_PINGRESP_TIMEOUT_MS should be same at all the places?

Reply:
Changed all the configs to use the same text as this file